web/pgadmin/tools/schema_diff/static/js/components/InputComponent.jsx

```diff
@@ -17,7 +17,7 @@ import { SchemaDiffEventsContext } from './SchemaDiffComponent';
 import { SCHEMA_DIFF_EVENT } from '../SchemaDiffConstants';
 
 
-export function InputComponent({ label, serverList, databaseList, schemaList, diff_type, selectedSid = null, selectedDid=null, selectedScid=null }) {
+export function InputComponent({ label, serverList, databaseList, schemaList, diff_type, selectedSid = null, selectedDid=null, selectedScid=null, onUpdateSelectedRowIds }) {
```

Comment:
onUpdateSelectedRowIds - the name itself is incorrect. InputComponent has no relation with selected row ids. It should be `onServerSchemaChange`. `onServerSchemaChange` should be called directly - `onServerSchemaChange()`. The caller of component should implement what to be done in `onServerSchemaChange`. Don't perform the business logic of one component in another.